src/View/Window.vala

```diff
@@ -342,6 +342,11 @@ namespace Marlin.View {
                 });
             });
 
+            tabs.tab_added.connect ((tab) => {
```

Comment:
When tabs are created other than by dropping them onto the notebook, window already knows about it.  Only on the implementation of drag/drop of tabs between windows was it necessary to pay attention to this signal so that the back reference from ViewContainer to window could be updated.